auth/src/main/res/layout/provider_button_email.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Button
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/email_button"
+    style="@style/FirebaseUI.Button.AccountChooser.EmailButton"
+    android:text="@string/sign_in_with_email"
+    tools:ignore="UnusedIds"/> <!-- Used in tests -->
```

Comment:
It took me a while to figure out that my build was failing because lint thinks these ids are unused so I moved the suppression from the `lint-baseline` to each button.